sklearn/model_selection/tests/test_successive_halving.py

```diff
@@ -410,16 +413,6 @@ def test_top_k(k, itr, expected):
     assert np.all(got == expected)
 
 
-def test_refit_callable():
```

Comment:
this test got deleted but shouldn't we test `_select_best_index` in the same way now?

Reply:
I added back the test here.